api/v1/model_handlers.go

```diff
@@ -272,6 +274,20 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Param(ws.QueryParameter("end", "End time for requested metric").DataType("string")).
 		Writes(MetricResult{}))
 
+	// The /sum/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} endpoint exposes
+	// an aggregated metric for a list od pods of the model.
+	ws.Route(ws.GET("/sum/namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name}").
```

Comment:
At the moment, /namespaces/{namespace-name}/pods/ returns a list of all the pods, along with the latest cpu and memory usage values. If we change this behavior, then the API will not be fully browsable, as there would be no way for a consumer to know which pods are represented in the model.